moped-editor/src/views/projects/projectView/ProjectNotes.js

```diff
@@ -329,21 +355,22 @@ const ProjectNotes = (props) => {
     <CardContent>
       <Grid container spacing={2}>
         {/*New Note Form*/}
-        {!editingNote && (
+        {!isEditingNote && (
           <Grid item xs={12}>
             <Card>
               <NoteInput
                 noteText={noteText}
                 setNoteText={setNoteText}
                 newNoteType={newNoteType}
                 setNewNoteType={setNewNoteType}
-                editingNote={editingNote}
+                isEditingNote={isEditingNote}
                 noteAddLoading={noteAddLoading}
                 noteAddSuccess={noteAddSuccess}
                 submitNewNote={submitNewNote}
                 submitEditNote={submitEditNote}
                 cancelNoteEdit={cancelNoteEdit}
                 isStatusEditModal={isStatusEditModal}
+                validator={isStatusUpdate ? validator : null}
```

Comment:
This felt a little awkward, but the validation needs to only apply when entering status updates. The NoteInput only knows enough to use the validator function if there is one passed in this prop.